lib/evaluation/evaluationClient.ts

```diff
@@ -34,16 +34,16 @@ export class EvaluationClient {
   private _results: HandlerCheckResult[];
   private _verifiableCredential: Partial<VerifiableCredential>[];
   private _presentationSubmission: Partial<PresentationSubmission>;
-  private _did: string;
+  private _dids: string[];
```

Comment:
Yesterday you told us there was o ticket for this. Since the effort is the same to change to a presentation or vcs + dids I decided the fix it right away.